README.md

```diff
@@ -654,7 +654,7 @@ MIT
 
 ## Catalog of apps and CLIs
```

Comment:
Everything below is automatically generated, I think we should fix these in the command descriptions, because those requirements will overwrite every change you made.